src/api/init.ts

```diff
@@ -149,6 +152,31 @@ export async function startApiServer(opts: {
     res.redirect(`/extended/v1/status`);
   });
 
+  app.use('/doc', (req, res) => {
+    // if env variable for API_DOCS_URL is given
+    if (apiDocumentationUrl) {
+      return res.redirect(apiDocumentationUrl);
+    } else if (!isProdEnv) {
+      // use local documentation if serving locally
+      const apiDocumentationPath = path.join(__dirname + '../../../docs/.tmp/index.html');
+      if (fs.existsSync(apiDocumentationPath)) {
+        return res.sendFile(apiDocumentationPath);
+      }
+
+      const docNotFound = {
+        error: 'Local documentation not found',
+        desc: 'Please run the command: `npm run build:docs` and restart your server',
```

Comment:
What do you suggest on this? 
Do you think adding `npm run build:docs` in the root `Dockerfile` [here](https://github.com/hirosystems/stacks-blockchain-api/blob/develop/Dockerfile#L8) will solve the problem? 
Plus, can you suggest a few use-cases where it might fail?

Reply:
Yep updating that file should work. 

Clarinet uses the docker image